debug.go

```diff
@@ -18,13 +19,17 @@ func (debug *debugging) Enable(b bool) {
 // Printf writes debug output.
 func (debug debugging) Printf(format string, args ...interface{}) {
 	if debug {
+		format = "ldap-trace: " + format
 		log.Printf(format, args...)
 	}
 }
 
 // PrintPacket dumps a packet.
 func (debug debugging) PrintPacket(packet *ber.Packet) {
 	if debug {
-		ber.PrintPacket(packet)
+		var b bytes.Buffer
+		ber.WritePacket(&b, packet)
+		textToPrint := "ldap-trace: " + b.String()
```

Comment:
Could we set `"ldap-trace: "` as a constant once?

Reply:
Thanks for suggestion! Done.